kong/router/transform.lua

```diff
@@ -696,32 +696,32 @@ end
 local uuid_generator = assert(uuid.factory_v5('7f145bf9-0dce-4f91-98eb-debbce4b9f6b'))
 
 
-local function sort_by_regex_or_length(path)
-  return is_regex_magic(path) or #path
-end
-
+-- group array-like table 'paths' to regex and length groups
+local path_groups = {}
+local path_groups_map = {}
+local function group_by_regex_or_length(paths)
```

Comment:
This function is not easy to understand, could we add some comments or examples for it?

Reply:
I added documentation about it.